### PR TITLE
573 possibility for a leader to edit status

### PR DESCRIPTION
**Pull request by thomas-rcv**
- Outsourced modal window in own file
- New link for leaders and Admins to edit participation states of users
- some more logic and logic changes

### CODE REVIEW

**Comment by thomas-rcv** [comment]
Problem fixed and moved file to module dates

File: adm_program/modules/lists/lists_show.php [hunk 1]
```diff
@@ -389,6 +402,7 @@
 
         // create html page object
         $page = new HtmlPage();
+        $page->enableModal();
```
Comment (Fasse) [inline adm_program/modules/lists/lists_show.php]: Only enable if $editUserStatus = true

**Comment by thomas-rcv** [comment]
You´re right. Modal is now only allowed with permission to edit.
Correct message should also be shown now without any changes, except for the comments.
This, I want to keep at this time to enable documentation by leaders and admins afterwards, if deadline exceeded.

Notice.:
Deadline exceeded is also working within the PR. My data build is as old as it can show it up directly.
I think no tests are needed for this.

**Comment by Fasse** [comment]
Everything works fine :)

You can merge.

**Pull request merged**